srt.go

```diff
@@ -46,6 +46,12 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 
 		// Line contains time boundaries
 		if strings.Contains(line, srtTimeBoundariesSeparator) {
+			// Return the wrong number of rows
+			if len(s.Lines) == 0 {
+				err = fmt.Errorf("error in the (%v) line", lineNum)
```

Comment:
Can you replace this line by `err = fmt.Errorf("astisub: line %d: no lines", lineNum)`?

Indeed we want to prefix errors as well as provide more information in the message.